jc/parsers/curl_head.py

```diff
@@ -0,0 +1,123 @@
+"""jc - JSON Convert `Curl` string parser
+
+Supports output of Curl when run with the HEAD option (-I)
+
+Usage (cli):
+
+    $ curl --head example.org | jc --curl-head
+
+Usage (module):
+
+    import jc
+    result = jc.parse('curl-head', curl_output)
+
+Schema:
+
+Curl output converted to a dictionary
+
+    {
+      "key1":       string,
+      "key2":       string
+    }
+
+Examples:
+
+curl -s -I google.com | python3 -m jc --curl-head | jq
+{
+  "Location": "http://www.google.com/",
+  "Content-Type": "text/html; charset=UTF-8",
+  "Content-Security-Policy-Report-Only": "object-src 'none';base-uri 'self';script-src 'nonce-XXXXXXXX' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp",
+  "Date": "Wed, 10 Jan 2024 19:12:42 GMT",
+  "Expires": "Fri, 09 Feb 2024 19:12:42 GMT",
+  "Cache-Control": "public, max-age=2592000",
+  "Server": "gws",
+  "Content-Length": "219",
+  "X-XSS-Protection": "0",
+  "X-Frame-Options": "SAMEORIGIN",
+  "HTTP-Version": "1.1",
+  "Status-Code": "301",
+  "Status-Reason": "Moved Permanently"
+}
+"""
+import jc.utils
+import configparser
+import re
+from jc.parsers import kv
+
+
+class info():
+    """Provides parser metadata (version, author, etc.)"""
+    version = '2.0'
+    description = 'curl head parser'
+    author = ''
+    author_email = ''
```

Comment:
Who wants to take the credit/blame for this parser? 😄

Reply:
fixed cca4ef4